grammar/es/src/rules.rs

```diff
@@ -112,34 +204,75 @@ pub fn rules_duration(b: &mut RuleSetBuilder<Dimension>) -> RustlingResult<()> {
                       |_| Ok(UnitOfDurationValue::new(Grain::Month))
     );
     b.rule_1_terminal("year (unit-of-duration)",
-                      b.reg(r#"a(?:n|ñ)os?"#)?,
+                      b.reg(r#"a[nñ]os?"#)?,
                       |_| Ok(UnitOfDurationValue::new(Grain::Year))
     );
     b.rule_1_terminal("quarter of an hour",
```

Comment:
tiny remark: you authorize "un 1/4 de hora", but I guess that does not create ambiguity so that should be fine